### ISSUE TITLE
Install failed Kodi 18.1 (coreELEC 9.0.1)

### ISSUE THREAD

**Issue by T0W1** [jellyfin/jellyfin-kodi#25]
install fails and it complains about folder structure.

**Comment by mcarlton00** [comment]
What install method were you using?  And was it the repository install that failed or the actual addon?

**Comment by T0W1** [comment]
The actual add-on. I downloaded the git-repo as a zip file, it dit install on kodi 17.6 on Ubuntu-mate 16.04.6.

BTW is there a kodi repo?

**Comment by mcarlton00** [comment]
A Kodi repo went up last week at https://repo.jellyfin.org/releases/client/kodi/ and the [installation docs](https://jellyfin.readthedocs.io/en/latest/clients/#installing) have been updated.

I've installed from the zip file several times and it's never complained that I can think of.  You tried to install the same zip file on both kodi 17 and 18, correct?  I suppose it's also possible that the file got corrupted when you downloaded it (lost packets or a dropped signal at the wrong time), which would explain why it errored on file structure.

Something else I've noticed is that if you uninstall the addon and then try to reinstall it again in the same session (without restarting kodi) it will error, but I don't remember the explanation it gave.  This may also occur with the emby for kodi addon, since they both do the same database manipulations.

Try installing again via the repo, and if it keeps having trouble somebody will have to look into coreelec more and see if they do anything different.
